core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2583,11 +2584,22 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_METASTORE_ROCKS_CHECKPOINT_COMPRESSION_LEVEL =
       intBuilder(Name.MASTER_METASTORE_ROCKS_CHECKPOINT_COMPRESSION_LEVEL)
-          .setDefaultValue(1)
+          .setDefaultValue(0)
```

Comment:
the description for this key probably needs to be updated

Reply:
done